rosbag2_bag_v2_plugins/src/rosbag2_bag_v2_plugins/storage/rosbag_v2_storage.hpp

```diff
@@ -43,6 +43,10 @@ class RosbagV2Storage : public rosbag2_storage::storage_interfaces::ReadOnlyInte
 
   rosbag2_storage::BagMetadata get_metadata() override;
 
+  uint64_t get_bagfile_size() const override;
+
+  std::string get_storage_identifier() const override;
```

Comment:
Method doc?

Reply:
Added method doc for `get_storage_identifier`